dojo-blog-postAPI-date/src/Input.js

```diff
@@ -0,0 +1,112 @@
+import { useHistory } from "react-router-dom";
+
+const initialValues = {
+  title: "",
+  body: "",
+  author: "mario",
+  date: "",
+  status: "completed",
+};
+const Input = () => {
+  const history = useHistory();
+  const { values, errors, touched, handleBlur, handleChange, handleSubmit } =
+    useFormik({
+      initialValues: initialValues,
+
+      validationSchema: signUpSchema,
+
+      onSubmit: (values, action) => {
+        formSubmitHandler(values);
+        action.resetForm();
+        console.log(values);
+        history.go(-1);
+      },
+    });
+  const queryClient = useQueryClient();
+
+  const onSuccess = () => {
+    queryClient.invalidateQueries("key");
+  };
+
+  const { mutate } = SetLoginFormFormik(onSuccess);
+
+  const formSubmitHandler = (values) => {
+    mutate(values);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="blog-details">
+        <label htmlFor="title">Title: </label>
+        <input
+          type="title"
+          name="title"
+          placeholder="title"
+          value={values.title}
+          onChange={handleChange}
+          onBlur={handleBlur}
+        />
+        {errors.title && touched.title ? (
+          <p className="form-errors">{errors.title}</p>
+        ) : null}
+      </div>
+      <div className="blog-details">
+        <label htmlFor="body">Body: </label>
+        <input
+          type="body"
+          name="body"
+          placeholder="Body"
+          value={values.body}
+          onChange={handleChange}
+          onBlur={handleBlur}
+        />
+        {errors.body && touched.body ? (
+          <p className="form-errors">{errors.body}</p>
+        ) : null}
+      </div>
+      <div className="blog-details">
+        <label>Blog author: </label>
+        <select name="author" value={values.author} onChange={handleChange}>
+          <option value="mario">mario</option>
+          <option value="yoshi">yoshi</option>
+        </select>
+      </div>
+
+      <div className="blog-details">
+        <label htmlFor="date">Date :</label>
+        <input
+          type="date"
+          name="date"
+          placeholder="Date"
+          value={values.date}
+          onChange={handleChange}
+          onBlur={handleBlur}
+        />
+        {errors.date && touched.date ? (
+          <p className="form-errors">{errors.date}</p>
+        ) : null}
+      </div>
+      <div className="blog-details">
+        <label>Status:</label>
+        <select name="status" value={values.status} onChange={handleChange}>
+          <option value="completed">completed</option>
+          <option value="pending">pending</option>
+        </select>
+
+        {errors.status && touched.status ? (
+          <p className="form-errors">{errors.status}</p>
+        ) : null}
+        <button className="input-button" type="submit">
+          Add blog
+        </button>
+      </div>
+    </form>
+  );
+};
+
+export default Input;
```

Comment:
Why name it as Input. Input is a reserved keyword in HTML
So please use some valida name like CreatBlogForm